### PR TITLE
Fix bz2 and openssl manpages.

### LINKED ISSUE #3002: Packages that contain files in /usr/local/man that need to be installed in /usr/local/share/man
- [ ] bz2
- [ ] openssl

### PR DESCRIPTION
**Pull request by superloach**
Fixes #3002.
OpenSSL will be updated separately, for now this just solves the manpage issue.

### CODE REVIEW

File: packages/openssl.rb [hunk 1]
```diff
@@ -52,8 +39,9 @@ def self.install
     system "make", "INSTALL_PREFIX=#{CREW_DEST_DIR}", "install"
     system "find #{CREW_DEST_PREFIX} -name 'lib*.a' -print | xargs rm"
 
-    # move man to #{CREW_PREFIX}/man
-    system "mv", "#{CREW_DEST_DIR}/etc/ssl/man", "#{CREW_DEST_PREFIX}/man"
+    # move man to #{CREW_PREFIX}/share/man
+    system "mkdir", "-p", "#{CREW_DEST_PREFIX}/share"
```
Comment (JL2210) [inline packages/openssl.rb]: Please change to `FileUtils.mkdir_p("#{CREW_DEST_PREFIX}/share")`.

File: packages/openssl.rb [hunk 2]
```diff
@@ -3,23 +3,10 @@
 class Openssl < Package
   description 'OpenSSL is an open source project that provides a robust, commercial-grade, and full-featured toolkit for the Transport Layer Security (TLS) and Secure Sockets Layer (SSL) protocols.'
   homepage 'https://www.openssl.org/'
-  version '1.0.2p'
+  version '1.0.2p-2'
```
Comment (uberhacker) [inline packages/openssl.rb]: This could be `version '1.0.2p-1'`.  No need to increment more than 1.

File: packages/bz2.rb [hunk 3]
```diff
@@ -49,6 +36,10 @@ def self.install
     system "ln -s libbz2.so.1.0.6 #{CREW_DEST_LIB_PREFIX}/libbz2.so.1.0"
     system "ln -s libbz2.so.1.0.6 #{CREW_DEST_LIB_PREFIX}/libbz2.so.1"
     system "ln -s libbz2.so.1.0.6 #{CREW_DEST_LIB_PREFIX}/libbz2.so"
+
+    # Move manpages
+    system "mkdir -p #{CREW_DEST_PREFIX}/share"
```
Comment (uberhacker) [inline packages/bz2.rb]: @JL2210: Why didn't suggest `FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share"` here also?

Comment (JL2210) [reply packages/bz2.rb]: Forgot about it. Sorry.

**Pull request merged**